src/frontend/package.json

```diff
@@ -47,17 +47,20 @@
     "webpack-cli": "4.3.1"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "5.15.1",
     "@types/emojione": "2.2.6",
+    "@types/highlightjs": "^10.1.0",
     "bootstrap": "4.5.3",
     "draft-js": "0.11.7",
     "draft-js-anchor-plugin": "2.0.3",
     "draft-js-buttons": "2.0.2",
+    "draft-js-code": "^0.3.0",
     "draft-js-emoji-plugin": "2.1.3",
     "draft-js-plugins-editor": "3.0.0",
     "draft-js-static-toolbar-plugin": "3.0.1",
     "emojione": "4.5.0",
     "es6-shim": "0.35.6",
-    "@fortawesome/fontawesome-free": "5.15.1",
+    "highlightjs": "^9.16.2",
```

Comment:
Is there a reason to not install the latest version ? `10.5.0`

Reply:
I didn't try to force it, I've let 

> yarn add 

decides for me. Is it a common use to overload the default configuration to force to add the latest version ?

Reply:
Whoops, I understood what's wrong.
You have installed `highlightjs` instead of `highlight.js` but in ` src/frontend/js/utils/highlight.ts` you are using `highlight.js`. May be it works locally if you have installed `highlight.js` during development then removed it of your dependencies?

Reply:
thanks, nice catch !